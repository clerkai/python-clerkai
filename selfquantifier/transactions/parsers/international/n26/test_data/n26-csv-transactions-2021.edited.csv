"Date","Payee","Account number","Transaction type","Payment reference","Amount (EUR)","Amount (Foreign Currency)","Type Foreign Currency","Exchange Rate"
"2021-05-01","Patreon* Membership","","MasterCard Payment","","-3.72","-3.72","EUR","1.0"
"2021-05-05","NETFLIX.COM","","MasterCard Payment","","-11.99","-11.99","EUR","1.0"
