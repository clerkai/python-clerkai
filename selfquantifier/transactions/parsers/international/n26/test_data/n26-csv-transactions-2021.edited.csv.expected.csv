Real Date,Bank Date,Payee,Bank Message,Amount,Currency,Balance,Foreign Currency,Foreign Currency Amount,Foreign Currency Rate,Original data,Raw Real Date,Raw Bank Date,Raw Payee,Raw Bank Message,Raw Amount,Raw Currency,Raw Balance,Raw Foreign Currency,Raw Foreign Currency Amount,Raw Foreign Currency Rate
,2021-05-01,Patreon* Membership,Patreon* Membership,-3.72,EUR,,EUR,-3.72,1.00000000000000,"{'Date': '2021-05-01', 'Payee': 'Patreon* Membership', 'Account number': nan, 'Transaction type': 'MasterCard Payment', 'Payment reference': nan, 'Amount (EUR)': -3.72, 'Amount (Foreign Currency)': -3.72, 'Type Foreign Currency': 'EUR', 'Exchange Rate': 1.0}",,2021-05-01,Patreon* Membership,,-3.72,,,EUR,-3.72,1.0
,2021-05-05,NETFLIX.COM,NETFLIX.COM,-11.99,EUR,,EUR,-11.99,1.00000000000000,"{'Date': '2021-05-05', 'Payee': 'NETFLIX.COM', 'Account number': nan, 'Transaction type': 'MasterCard Payment', 'Payment reference': nan, 'Amount (EUR)': -11.99, 'Amount (Foreign Currency)': -11.99, 'Type Foreign Currency': 'EUR', 'Exchange Rate': 1.0}",,2021-05-05,NETFLIX.COM,,-11.99,,,EUR,-11.99,1.0
